common/jenkins-agents/python/docker/Dockerfile

```diff
@@ -5,44 +5,43 @@ LABEL maintainer="Gerard Castillo <gerard.castillo@boehringer-ingelheim.com>"
 ARG nexusHost
 ARG nexusAuth
 
-ENV PYTHON_VERSION=3.6.10
-ENV VENV_VERSION=20.0.9
-ENV INSTALL_PKGS="yum-utils gcc make openssl-devel zlib-devel sqlite-devel"
-
-RUN set -x \
-    && yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS \
-    && yum clean all \
-    && rm -rf /var/cache/yum/*
-
-RUN cd /tmp \
-    && curl -O https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tgz \
-    && tar xzf Python-${PYTHON_VERSION}.tgz -C / \
-    && rm -rf Python-${Python_VERSION}.tgz
-
-RUN cd /Python-${PYTHON_VERSION} \
-    && ./configure \
-    && make altinstall \
-    && ln -s /Python-${PYTHON_VERSION}/python /usr/local/sbin/python3 \
-    && python3 -V \
-    && chmod a+rx /Python-${PYTHON_VERSION} \
-    && chmod a+rx /Python-${PYTHON_VERSION}/python \
-    && yum remove -y $INSTALL_PKGS
-
-RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" \
-    && python3 get-pip.py
+ENV LANGUAGE en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+ENV LC_CTYPE en_US.UTF-8
+ENV LC_MESSAGES en_US.UTF-8
+ENV PATH=/opt/rh/rh-python36/root/usr/bin${PATH:+:${PATH}} \
+    LD_LIBRARY_PATH=/opt/rh/rh-python36/root/usr/lib64${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}} \
+    MANPATH=/opt/rh/rh-python36/root/usr/share/man:$MANPATH \
+    PKG_CONFIG_PATH=/opt/rh/rh-python36/root/usr/lib64/pkgconfig${PKG_CONFIG_PATH:+:${PKG_CONFIG_PATH}} \
+    XDG_DATA_DIRS="/opt/rh/rh-python36/root/usr/share:${XDG_DATA_DIRS:-/usr/local/share:/usr/share}" \
+    BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
+
+COPY contrib/bin/scl_enable /usr/local/bin/scl_enable
+
+# Enable rhel-server-rhscl-7-rpms repo
+RUN set -x && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --disable rhel-7-server-htb-rpms && \
+    yum makecache
```

Comment:
Will this not fail on CENTOS?

Reply:
I tested builds within openshift and locally building from the main jenkins agent base and it works :) it was already working before within the airflow agent in ODS2.x.
What I want to improve anyway is the usage of the scl_enable script, I think it might not be used, but I just got inspired from the nodejs agent one :)

Reply:
Ah nice. Talk to @oalyman regarding this `scl_enable` thingy, I remember he had questions about it when updating the Scala agent, and might have removed it completely.